apps/address-book/app/app-state-reducer.js

```diff
@@ -0,0 +1,7 @@
+function appStateReducer(state) {
+  return {
+    ...state,
```

Comment:
Given that the `reducer` prop to `AragonApi` is optional, does it benefit us to have a no-op reducer at this point? Can we get rid of this whole file?

Reply:
Yes. I guess there is no point in keeping code like that for future use - it can always be added when it's needed...